src/main/java/uk/gov/companieshouse/api/accounts/controller/LoansController.java

```diff
@@ -0,0 +1,151 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import uk.gov.companieshouse.api.accounts.AttributeName;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.model.rest.smallfull.notes.loanstodirectors.Loan;
+import uk.gov.companieshouse.api.accounts.model.rest.smallfull.notes.loanstodirectors.LoansToDirectors;
+import uk.gov.companieshouse.api.accounts.service.impl.LoanServiceImpl;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
+import uk.gov.companieshouse.api.accounts.utility.LoggingHelper;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+
+@RestController
+@RequestMapping("/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/notes/loans-to-directors/loans")
+public class LoansController {
+
+    @Autowired
+    private LoanServiceImpl loanService;
+
+    @Autowired
+    private ApiResponseMapper apiResponseMapper;
+
+
+    @PostMapping
+    public ResponseEntity create(@RequestBody Loan loan, @PathVariable("companyAccountId") String companyAccountId,
+    		HttpServletRequest request) {
+
+        Transaction transaction = (Transaction) request.getAttribute(AttributeName.TRANSACTION.getValue());
+
+        try {
+            ResponseObject<Loan> response =
+                    loanService.create(loan, transaction, companyAccountId, request);
+
+            return apiResponseMapper
+                    .map(response.getStatus(), response.getData(), response.getErrors());
```

Comment:
Shouldn't this be logged?

Reply:
Everything is logged in an interceptor (request and response) so logging in the controller isn't necessarily needed, but that's a preference thing